<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QMainWindow" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>160</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addRUC">
       <property name="text">
        <string>add new RUC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editRUC">
       <property name="text">
        <string>edit exist RUC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showOnt">
       <property name="text">
        <string>show Ontology</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="ontologyPathLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>null</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
